app/models/metric/ci_mixin/processing.rb

```diff
@@ -98,24 +94,12 @@ def perf_process(interval_name, start_time, end_time, counters_data)
       publish_metrics(rt_rows) if syndicate_metrics?
     end
     _log.info("#{log_header} Processing for #{log_specific_targets(resources)}, for range [#{start_time} - #{end_time}]...Complete - Timings: #{t.inspect}")
-
-    affected_timestamps
   end
 
   private
 
-  def transform_resources!(counters_data)
-    # Fetch ActiveRecord object by 1 query per Model
-    grouped_resource_refs = counters_data.keys.each_with_object({}) { |x, obj| (obj[x.first] ||= []) << x.second }
-    fetched_records = grouped_resource_refs.keys.each_with_object({}) do |x, obj|
-      x.constantize.where(:id => grouped_resource_refs[x]).each { |rec| obj[[x, rec.id]] = rec }
-    end
-    # Transforming [Class, id] that were sent via the counters_data into the ActiveRecord objects
-    counters_data.transform_keys! { |x| fetched_records[x] }
-  end
```

Comment:
We were sending `[class.name, id]` over the wire to this method.
I'm guessing that was to support sending these values over kafka to have a single point of saving.

We can bring this back if we decide to separate fetching of metrics with saving them.

Reply:
It looks like per 7eebbeb7dcd it was `# Convert to format allowing to send multiple resources at once` not really sure how that helps with that though :shrug: